sites/automationworld/server/routes/content.js

```diff
@@ -2,7 +2,7 @@ const { withContent } = require('@base-cms/marko-web/middleware');
 const content = require('../templates/content');
 const company = require('../templates/content/company');
 const webinar = require('../templates/content/webinar');
-const queryFragment = require('../graphql/fragments/content-page');
+const queryFragment = require('../../../../packages/common/graphql/fragments/content-page');
```

Comment:
I realize the companyQuery import is also "wrong" so this isn't your fault. :) However... instead of relatively resolving the `queryFragment` and the `companyQueryFragment`, resolve them using the package name. So:

```
const queryFragment = require('@base-cms-websites/package-common/graphql/fragments/content-page');
```

and...
```
const companyQueryFragment = require('@base-cms-websites/package-common/graphql/fragments/content-company');
```

Change in all content routes please :)